cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -26,6 +28,8 @@ const Modules = () => {
   const [allData, setAllData] = useState([]);
   const [filteredData, setFilteredData] = useState([]);
   const [loadingModules, setLoadingModules] = useState(false);
+  const [displaymodule, setDisplayModule] = useState<string[]>([]);
```

Comment:
Can you rename this to `moduleHealthFilter`?

Reply:
Done